habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/util/ContainerizeDepsDockerfileContext.java

```diff
@@ -0,0 +1,58 @@
+package org.technologybrewery.habushu.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.velocity.VelocityContext;
+
+public class ContainerizeDepsDockerfileContext extends VelocityContext {
```

Comment:
A: Docker Context has a very specific meaning, and this name doesn't indicate it's for use with Velocity. 
```suggestion
public class ContainerizeDepsVelocityContext extends VelocityContext {
```